adal/src/androidTest/java/com/microsoft/aad/adal/StorageHelperTests.java

```diff
@@ -249,6 +250,9 @@ public void testVersion() throws GeneralSecurityException, IOException {
         }
     }
 
+
+    //Github issue #580. Suppress this unit test as we cannot make it work consistently.
+    @Suppress
```

Comment:
I remember we've discussed about this unit test before and decided to suppress it for now.